plugins/package-managers/node/src/funTest/assets/projects/synthetic/npm/babel/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "npm-project-that-depends-on-babel",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
For completeness, could you add to the commit message which version of NPM was used to recreate the lockfile? I noticed that the lockfile format version changed, but I guess that's ok as this specific test does not depend on it.